additional-features.md

```diff
@@ -43,3 +53,285 @@ Or a `422 Unprocessably Entity` status code, with an error description in the bo
 ```
 
 Once you have the account title you can compare that with the recipient details you wish to provide us, and only create a transaction if they match.
+
+## Collections from senders
+
+The BitPesa API can also be used to collect money from various senders. We currently support the following automated ways to collect money from the user:
+
+* GHS mobile collections
+* TZS mobile collections
+* UGX mobile collections
+* NGN mobile collections
+* NGN card collections
+* GBP bank collections through IBAN
+* EUR bank collections through IBAN
+
+The collected money can then be used to fund any of the an automated payout types (see the [transaction flow](transaction-flow.md)), or it can be used to fund a bank account. In the latter case payments will be aggregated and only sent out manually by our team once requested.
+
+How to handle automated collection requests are based on the particular currency and type of collection.
+
+You can also check the [API reference documentation](https://api.bitpesa.co/documentation#fetching-possible-payin-methods) for more details.
+
+The generic structure of a transaction that collects the equivalent of 100 USD from senders and pays out to an USD bank account (that will be payed out manually by our team in bulk) looks like the following:
+
+```javascript
+{
+  "input_currency": "NGN",
+  "payin_methods": [
+    // payin method details
```

Comment:
For clarity that this needs to be a hash

```suggestion
    {
      // payin method details
    }
```

Reply:
It's both a hash and an array just to make matters worse. I made it so in the examples below the content is fully featured to make it explicit, but you're right I'll add it here as well